run/awake_start

```diff
@@ -78,13 +84,23 @@ case "${TESLA_WAKE_MODE:-stream}" in
     is_sentry_mode_enabled=$(/root/bin/tesla_api.py is_sentry_mode_enabled | tr '[:upper:]' '[:lower:]')
     if [ "false" = "${is_sentry_mode_enabled}" ]
     then
-      log "Temporarily enabling Sentry Mode to keep car awake."
-      touch /tmp/disable_sentry_after_archiving
-      /root/bin/tesla_api.py enable_sentry_mode &>> "${LOG_FILE}"
+      # Check if TESLAFI_WAKE_MODE is set to sentry
+      if [ "$TESLAFI_WAKE_MODE" == "sentry" ]; then
+        log "Turning on Sentry Mode to keep car awake (via TeslaFI)."
+        set_sentry_mode true
+        echo "sentry_enabled" > /tmp/sentry_state  # Log Sentry state
+      else
+        log "Turning on Sentry Mode to keep car awake (via Tesla API)."
+        /root/bin/tesla_api.py enable_sentry_mode &>> "${LOG_FILE}"
+        echo "sentry_enabled" > /tmp/sentry_state  # Log Sentry state
+      fi
+    else
+      log "Sentry Mode is already enabled."
+      echo "sentry_already_enabled" > /tmp/sentry_state  # Log Sentry state
     fi
     ;;
   stream)
-    log "Starting background task to keep car awake."
+    log "Starting background tasks to keep TeslaUSB powered on during archiving." 
```

Comment:
Why did you change this log message, and specifically why did you pluralize "task" to "tasks" when there's only one background task?